setup.py

```diff
@@ -28,33 +28,15 @@
     'pytest>=3.8.1,<4',
 ]
 
-extras_require = {
```

Comment:
The `extras_require` thingy, we can keep since we need it in order to be able e.g. to do something like `pip install -e .[docs,tests]` when we want to run only the tests and build the documentation of the project.

What I meant in the previous comment is to remove the `invenio-accounts` dependency (since it's not used by anyone)